blocks/extras/extensions/plot.py

```diff
@@ -44,12 +44,16 @@ class Plot(SimpleExtension):
     document : str
         The name of the Bokeh document. Use a different name for each
         experiment if you are storing your plots.
-    channels : list of lists of strings
+    channels : list of lists of strings or dicts
```

Comment:
This doesn't read. I had to read code to understand that it means `list of list of strings OR list of dicts`.